docs/src/references/agent/restart-operation.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Restart Operation
+tags: [Reference, Agent, Restart]
+sidebar_position: 4
+---
+
+# Restart Operation
+
+Thin-edge defines a `restart` operation to restart a device, being the main device or a child device.
+
+- A restart is typically triggered by a [mapper](../mappers) on behalf of a cloud operator.
+- A restart can also be triggered from another operation (as a software update)
+  or service (detecting for instance some anomalies requesting a reboot).
+- `tedge-agent` is the reference implementation of the `restart` operation.
+- However, a custom `restart` plugin implementation can be installed on a device with specific requirements.
```

Comment:
This sentence can be a little misleading as it is not 100% clear that you need to display the restart function handled by the tedge-agent before doing your own implementation (assuming you still want to use other aspects of the tedge-agent, just not the restart functionality).

Reply:
I don't understand what you mean by *need to display*.

Reply:
Sorry it was a type/word-mixup....it should read `need to disable`

Reply:
I will add a note to clarify this point.